modules/auxiliary/scanner/redis/unauth_file_upload.rb

```diff
@@ -0,0 +1,130 @@
+##
```

Comment:
I'll remove this file when I land.  It looks like maybe I used the wrong name for the module when I sent you these PRs, so now we have two -- the old original one (this one, unauth_file_upload.rb) and newer one.